precompiles/ArbSys.go

```diff
@@ -114,9 +114,20 @@ func (con *ArbSys) SendTxToL1(c ctx, evm mech, value huge, destination addr, cal
 	if err != nil {
 		return nil, err
 	}
+	arbosState := c.State
+
+	if arbosState.ArbOSVersion() >= params.ArbosVersion_41 && value.BitLen() != 0 {
+		numOwners, err := arbosState.NativeTokenOwners().Size()
+		if err != nil {
+			return nil, err
+		}
+		if numOwners > 0 {
+			return nil, errors.New("not allowed to send value when native token owners exist")
```

Comment:
nitpick: how about adding a comment here explaining why this is not allowed?

Reply:
Done.